edgexfoundry-holding/device-onvif-camera
# refactor!: Replace internal topics from config with new constants

BREAKING CHANGE: Internal topics no longer configurable, except the base topic.







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md


## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Run non-secure EdgeX stack
Build and run device service locally with `-cp` (kill and restart for hybrid mode)
Verify the log has on following start-up :
```
msg="Subscribing to command requests on topic: edgex/device/command/request/device-onvif-camera/#"
msg="Responses to command requests will be published on topic: edgex/response/device-onvif-camera/<requestId>"
"Subscribing to System Events on topic: edgex/system-events/core-metadata/+/+/device-onvif-camera/#
```

## New Dependency Instructions (If applicable)
